build/build/src/ci_gen/job.rs

```diff
@@ -546,13 +546,18 @@ impl JobArchetype for PackageIde {
             } else {
                 shell(TEST_COMMAND)
             };
-            let test_step = test_step
+            let mut test_step = test_step
                 .with_env("DEBUG", "pw:browser log:")
                 .with_secret_exposed_as(secret::ENSO_CLOUD_TEST_ACCOUNT_USERNAME, "ENSO_TEST_USER")
                 .with_secret_exposed_as(
                     secret::ENSO_CLOUD_TEST_ACCOUNT_PASSWORD,
                     "ENSO_TEST_USER_PASSWORD",
                 );
+            // Make E2E tests optional on Windows, as we have an ongoing issue with the runner.
+            // TODO[ib]: remove once the issue is resolved.
+            if target.0 == OS::Windows {
```

Comment:
Isn't:
```yaml
continue-on-error: true
```
in the `gui.yaml` file a sufficient fix?

Reply:
This is the generator source that outputs `guy.yml` file.

Reply:
I see. I was blind.